### ISSUE TITLE
Few Bugs

### ISSUE THREAD

**Issue by mobbdeep** [Contentify/Contentify#156]
Hello,

Here is a list of bugs I have noticed..

- Creating a news article; once you press submit, the following error appears. The article is created, but when you click on the preview to access the article, the following errors appears. Also, same error when deleting the article.

> Whoops, looks like something went wrong.
> 1/1
> FatalErrorException in SimpleXMLElement.php line 6:
> Class 'SimpleXMLElement' not found
> in SimpleXMLElement.php line 6

- Suggestions:

1. Ability to remove sidebar content if not being used: Eg. Lastest Matches, Cups, etc.
2. Ability to add/remove users. Eg. If I want to register a team member myself, I could do so.
3. Ability to remove navigation options.
4. Ability to modify website title via back-end.

Thank you!

**Comment by chriskonnertz** [comment]
Hello,

> Creating a news article; once you press submit, the following error appears

Maybe a PHP extension is missing, take a look at this post for more details and a proposal for a solution: http://stackoverflow.com/questions/35223926/class-simplexmlelement-not-found-on-puphpet-php-5-6#answers-header

> Ability to remove sidebar content if not being used: Eg. Lastest Matches, Cups, etc.

Just hiding it would be okay as a solution?

> Ability to add/remove users. Eg. If I want to register a team member myself, I could do so.

_Adding_: This will on my list as well, will be implemented in the future.
_Removing_: This is not possible and never will be. Maybe we can create a workaround but actually removing user accounts is not an option.

> Ability to remove navigation options.

What you mean is "ability to remove items from the frontend main navigation", right?

> Ability to modify website title via back-end

Good idea, this most likely will be implemented in the future.

Thanks for your ideas! :)

**Comment by mobbdeep** [comment]
Hi Chris,

Thank you for the first solution to resolving the news article posting. I am indeed using Ubuntu with PHP7 therefore I needed the XML extension added. All is working well with that!

> Just hiding it would be okay as a solution?

Yes, perfect. I didn't mean removing it permanently, more or so the ability to hide/show sections that may not get used.

>What you mean is "ability to remove items from the frontend main navigation", right?

Yes, this is correct. Just a simple show/hide and the ability to add your own navigation options via back-end possibly?

One other thing I wanted to know is how can I set the back-end URL to run off my actual domain and not the server address. I did install it using the server IP address as per installation instructions, but the front-end runs off my domain as normal unless something is clicked within the back-end admin panel that then opens the website; Eg. "Website." I do not want my server IP address visible due to possible risks to attacks and whatnot.
